tests/unit/neptune/new/types/atoms/test_file.py

```diff
@@ -166,6 +165,8 @@ def test_stream_read_once(self):
             _ = file.content
 
     def test_as_image(self):
+        from PIL import Image
+
```

Comment:
shouldn't PIL be loaded using `pytest.importskip` here too?

Reply:
Good point!